src/gui/application.cpp

```diff
@@ -95,15 +95,6 @@ Application::Application(Platform *platform, const QString &displayLanguage, boo
         AbstractNetworkJob::httpTimeout = cfg.timeout();
     }
 
-    // Check vfs plugins
-    if (Theme::instance()->showVirtualFilesOption() && VfsPluginManager::instance().bestAvailableVfsMode() == Vfs::Off) {
```

Comment:
I think we should log what kind of plugin is loaded. Does that happen elsewhere?

Reply:
line 79